### PR TITLE
add helm notes

### PR DESCRIPTION
**Pull request by yagonobre**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/326)

### CODE REVIEW

File: helm/chart/teresa/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+To create a admin user, run the below commands:
+
```
Comment (aguerra) [inline helm/chart/teresa/templates/NOTES.txt]: Typo: `an admin`

Comment (yagonobre) [reply helm/chart/teresa/templates/NOTES.txt]: thanks :)

File: helm/chart/teresa/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  1. Download the client on https://github.com/luizalabs/teresa/releases
+    If you use osx you can install with homebrew, running:
+
+    brew tap luizalabs/teresa-cli
+    brew install teresa
+
+  2. Get the service external IP address by: kubectl get svc --namespace {{ .Release.Namespace }} -l app={{ template "fullname" . }} -o wide
+
+  3. Configure the Teresa client
+    teresa config set-cluster <cluster-name> --server <cluster-endpoint>
```
Comment (drgarcia1986) [inline helm/chart/teresa/templates/NOTES.txt]: maybe add `--current`?

Comment (yagonobre) [reply helm/chart/teresa/templates/NOTES.txt]: looks better :)

**Pull request merged**